### PR TITLE
Andrew goenner

### PR DESCRIPTION
**Pull request by drewgoenner**
I completed the project, made a resume for my dog.  I linked it to my portfolio and brought in a google font. I made several mix-ins and variables.

### CODE REVIEW

File: less/index.less [hunk 1]
```diff
@@ -57,3 +57,285 @@ html, body {
     height: 100%;
 }
 
+@header-font: 'Lobster', cursive;
+@fav-color:green;
+
```
Comment (thereactgirl) [inline less/index.less:62]: Nice use of variables!

File: less/index.less [hunk 2]
```diff
@@ -57,3 +57,285 @@ html, body {
...
+
+h1 {
+	.header();
+}
+body{
+
+
+.container {
+	@media screen and (max-width: 500px){
```
Comment (thereactgirl) [inline less/index.less:107]: I think you may be nesting a bit too deep. I personally wouldn't want to nest more than three deep. You don't need to nest global styles either. Yes, it is syntactically correct and you could nest EVERYTHING inside your container, buuuutttt this actually makes it harder for other developers to follow along your code.

File: less/index.less [hunk 3]
```diff
@@ -57,3 +57,285 @@ html, body {
...
+
+.historypics(){
+	width: 25%;
+	margin: 1%;
+
+	img{
+	width: 90%;
+	}
+}
+
```
Comment (thereactgirl) [inline less/index.less:86]: This a GREAT use case of mixins! Nice work!

**Review by thereactgirl — COMMENTED** [review]
Great job Andrew!

File: less/index.less (hunk 2 above)
Comment (drewgoenner) [reply less/index.less:107]: Thanks for the advice!  I got so into the coding, I forgot not to nest past grandchildren.  I'll keep a closer eye out in the future!

**Pull request merged**